### ISSUE TITLE
Branch point identification

### ISSUE THREAD

**Issue by charliex210** [KrishnaswamyLab/PHATE#2]
Hello,
I admire your new method "PHATE" on learning the manifold structure of high dimension data, and I have tried it on our own single-cell RNA-seq dataset. I wonder would you release the code of branch point identification algorithm mentioned in your preprint.
Thanks!

**Comment by kmoon3** [comment]
Hi,
Thanks for your interest in PHATE! We do plan to release the branch point identification algorithm. However, it will probably take a month or two before the code is ready for distribution. Do you have a language preference?

**Comment by charliex210** [comment]
It would be nice if you can release the algorithm! For me both python and R are OK.
Thanks!

**Comment by chlee-tabin** [comment]
Hi I am trying out PHATE too and wondering whether there is any code snippet to identify branch points. Without it it is difficult to identify genes at each branches etc. Even snippets of code used to generate some of the figures of the bioarxiv paper would be helpful!

**Comment by sameet** [comment]
Yes, this will be very useful to have, and will make PHATE ultimately more useful.

**Comment by jfouyang** [comment]
Hi,
I am really impressed with the method and the PHATE embeddings looks really good with my own scRNA-seq dataset. I was wondering if there will be any updates soon to include branch point identification as that will be really useful for my analysis.

And thanks again for the great work!
